cockatrice/src/main.cpp

```diff
@@ -114,6 +113,21 @@ int main(int argc, char *argv[])
     translationPath = qApp->applicationDirPath() + "/../share/cockatrice/translations";
 #endif
 
+    QCommandLineParser parser;
+    parser.setApplicationDescription("Cockatrice");
+    parser.addHelpOption();
+    parser.addVersionOption();
+
+    parser.addOptions(
+        {{{"c", "connect"}, QCoreApplication::translate("main", "Connect on startup"), "user:pass@host:port"},
+         {{"d", "debug-output"}, QCoreApplication::translate("main", "Debug to file")}});
+
+    parser.process(app);
```

Comment:
Should we error check to ensure this works?
```cpp
if (!parser.parse(QCoreApplication::arguments())) {
        *errorMessage = parser.errorText();
        return CommandLineError;
    }
```

Reply:
.process() takes care of this, exiting and printing an error message if there is a problem.